src/components/manage/Controlpanels/Controlpanel.jsx

```diff
@@ -43,6 +44,66 @@ const messages = defineMessages({
   },
 });
 
+// Filters props.controlpanel.schema to only valid/relevant fields
+const filterSchema = (controlpanel) => {
+  const panelType = controlpanel['@id'].substring(
+    controlpanel['@id'].lastIndexOf('/') + 1,
+  );
+
+  const unwantedSettings = {
+    language: ['display_flags', 'always_show_selector'],
+    // search: ['enable_livesearch'],
+    // site: [
+    //   'display_publication_date_in_byline',
+    //   'icon_visibility',
+    //   'thumb_visibility',
+    //   'no_thumbs_portlet',
+    //   'no_thumbs_lists',
+    //   'no_thumbs_summary',
+    //   'no_thumbs_tables',
+    //   'thumb_scale_portlet',
+    //   'thumb_scale_listing',
+    //   'thumb_scale_table',
+    //   'thumb_scale_summary',
+    //   'toolbar_position',
+    //   'toolbar_logo',
+    //   'default_page',
+    // ],
+    // editing: ['available_editors', 'default_editor', 'ext_editor'],
+    // imaging: [
+    //   'highpixeldensity_scales',
+    //   'quality_2x',
+    //   'quality_3x',
+    //   'picture_variants',
+    //   'image_captioning',
+    // ],
```

Comment:
@danalvrz Let's go ahead and uncomment these, if it is working correctly when you test locally.